loginPrcess.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require 'connection.php';
+
+$userName = $_POST["userName"];
+$password = $_POST["password"];
+
+
+if (empty($userName) || empty($password)) {
+    echo "Please fill the required areas...!";
+} else {
+    $loginResultSet1 = Database::search("SELECT * FROM `traveler` WHERE `email`='" . $userName . "' AND `password`='" . $password . "'");
+    $loginNumRows1 = $loginResultSet1->num_rows;
+
+    if ($loginNumRows1 == 1) {
+        $loginData1 = $loginResultSet1->fetch_assoc();
```

Comment:
Unwanted code line. Remove this

Reply:
Not an unwanted line. Both Users and Staff users login to the system using same login page. Here, first check the traveler relation whether the credential are available or not. If the result is true, then execute the code lines inside this ' if ' case. If the matching record could not find in the traveler relation, then find them in the ' staff_member_new ' relation.